app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -1035,6 +1047,42 @@ class HomeFragment : Fragment() {
         view?.sessionControlRecyclerView?.adapter?.notifyDataSetChanged()
     }
 
+    private fun showPrivacyPopWindow() {
```

Comment:
Let's move this to a separate helper class so we add more things into this home fragment class.